.azure-pipelines/pipelines.yml

```diff
@@ -253,7 +253,7 @@ stages:
   jobs:
   - job: release
     timeoutInMinutes: 120
-    pool: x64-large
+    pool: envoy-x64-large
```

Comment:
where we can know the pool  name is changed?

Reply:
hmm - good q - @mum4k and i changed them at the end of last week - probably you could figure that info out from https://github.com/envoyproxy/ci-infra/ - its not something that should change too often

Reply:
relatedly - we are looking at separating some of the pools, so i will most likely make this dynamic soon